config/minimalist-templates/popups_templates.yaml

```
@@ -0,0 +1,531 @@
+                              box-shadow: none !important;
+                            }
+                            .off {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                            .on {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                    # item2: >
+                    #   [[[
+                    #     var bri = states[entity.entity_id].attributes.brightness;
+                    #     return Math.round(bri ? (bri/256*100) : '0') + '%';
+                    #   ]]]
+                #####
+                color:
+                  type: custom:light-entity-card
+                  entity: "[[[ return entity.entity_id ]]]"
+                  persist_features: true
+                  brightness: false
+                  color_temp: false
+                  white_value: false
+                  hide_header: true
+                  effects_list: false
+                  card_mod:
+                    style:
+                      ".control":
+                        $: |
+                          #interactionLayer{ z-index: 1; filter: opacity(0.6);}
+                          #backgroundLayer{
+                            filter: blur(20px);
+                          }
+                      .: |
+                        ha-card {
+                          background: none !important;
+                          box-shadow: none !important;
+                          padding: 10px 0 10px 0 !important;
+                        }
+          item3:
+            card:
+              type: custom:button-card
+              template: popup_list_items
+              custom_fields:
+                item1:
+                  card:
+                    icon: "mdi:lightbulb-on"
+                    name: Luminosité
+                    type: "custom:button-card"
+                    template: popup_button
+                    tap_action:
+                      action: navigate
+                      navigation_path: "#brightness"
+                    state:
+                      - operator: template
+                        value: >
+                          [[[ return new URL(window.location.href).hash !== '#color' ]]]
+                        styles:
+                          icon:
+                            - color: "rgba(var(--color-blue),1)"
+                          img_cell:
+                            - background-color: "rgba(var(--color-blue), 0.2)"
+                item2:
+                  card:
+                    icon: "mdi:palette"
+                    name: Couleur
+                    type: "custom:button-card"
+                    template: popup_button
+                    tap_action:
+                      action: navigate
+                      navigation_path: "#color"
+                    state:
```

```suggestion
                    state:
                      - operator: template 
                        value: >
                          [[[ 
                            var color_modes = ["hs", "rgb", "rgbw"];
                            return !entity.attributes.supported_color_modes.some(r => color_modes.includes(r));
                          ]]]
                        styles:
                          card:
                            - display: none
```
Do not display the color wheel when the light does not support it